<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>499</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QWidget" name="widget_main" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="maximumSize">
             <size>
              <width>230</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="title">
                <string>调试信息</string>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>16</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <property name="spacing">
                 <number>2</number>
                </property>
                <item row="3" column="1">
                 <widget class="QCheckBox" name="checkBox">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>同时打印两张二维码</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1" colspan="2">
                 <widget class="QPlainTextEdit" name="plainTextEdit">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" rowspan="3">
                 <widget class="QGroupBox" name="groupBox_5">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>97</height>
                   </size>
                  </property>
                  <property name="title">
                   <string/>
                  </property>
                  <widget class="QLabel" name="label">
                   <property name="geometry">
                    <rect>
                     <x>10</x>
                     <y>30</y>
                     <width>54</width>
                     <height>12</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>起始数字</string>
                   </property>
                  </widget>
                  <widget class="QLabel" name="label_2">
                   <property name="geometry">
                    <rect>
                     <x>10</x>
                     <y>70</y>
                     <width>31</width>
                     <height>16</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>间隔</string>
                   </property>
                  </widget>
                  <widget class="QLabel" name="label_3">
                   <property name="geometry">
                    <rect>
                     <x>90</x>
                     <y>70</y>
                     <width>31</width>
                     <height>16</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>打印</string>
                   </property>
                  </widget>
                  <widget class="QLabel" name="label_4">
                   <property name="geometry">
                    <rect>
                     <x>190</x>
                     <y>70</y>
                     <width>16</width>
                     <height>16</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>份</string>
                   </property>
                  </widget>
                  <widget class="QLineEdit" name="textEdit_2">
                   <property name="geometry">
                    <rect>
                     <x>40</x>
                     <y>69</y>
                     <width>51</width>
                     <height>21</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>1</string>
                   </property>
                  </widget>
                  <widget class="QLineEdit" name="textEdit_3">
                   <property name="geometry">
                    <rect>
                     <x>120</x>
                     <y>69</y>
                     <width>61</width>
                     <height>21</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>1</string>
                   </property>
                  </widget>
                  <widget class="QLineEdit" name="textEdit">
                   <property name="geometry">
                    <rect>
                     <x>60</x>
                     <y>20</y>
                     <width>141</width>
                     <height>31</height>
                    </rect>
                   </property>
                   <property name="font">
                    <font>
                     <family>微软雅黑</family>
                     <pointsize>12</pointsize>
                     <weight>50</weight>
                     <bold>false</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>DD5411300001</string>
                   </property>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>二维码显示</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="rencode_view">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string> QR Code</string>
                 </property>
                 <property name="scaledContents">
                  <bool>false</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="rencode_view_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string> QR Code</string>
                 </property>
                 <property name="textFormat">
                  <enum>Qt::RichText</enum>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLineEdit" name="rencode_lineEdit">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="rencode_lineEdit_2">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="print_button">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>打印</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
